packages/universal-components/src/Checkbox/Checkbox.web.js

```diff
@@ -38,6 +38,7 @@ export default function Checkbox({
         name={name}
         checked={checked}
         style={styleInput}
+        readOnly
```

Comment:
I don't understand. Why it should be `readOnly` on the web? What for is Checkbox if you can't update its value? Or is it for something else?

Reply:
Because this input is actually hidden and all switching stuff is handled by the platform shared rest of the component.
It is there just to have valid html output on web.

Reply:
OK, I understand now. Thanks 👍